content/build/building/variables.md

```diff
@@ -343,7 +343,7 @@ Setting `NO_COLOR` to anything turns off colorized output, as recommended by
 
 ### BUILDKIT_HOST
 
-_Introduced in [Buildx v0.9.0](../release-notes.md#090)_
+{{< introduced buildx 0.9.0 "../release-notes.md#090" >}}
```

Comment:
This is not rendered because 0.9.0 is below the threshold set in `site.Params.min_version_thresholds.buildx`

Also generates a warning: https://github.com/docker/docs/actions/runs/7974560715/job/21770734246?pr=19452#step:4:301